src/login/oidc/oidcHandlers/AuthorizationCodeWithPkceOidcHandler.ts

```diff
@@ -52,6 +52,7 @@ export default class AuthorizationCodeWithPkceOidcHandler
   }
 
   async handle(oidcLoginOptions: IOidcOptions): Promise<ISolidSession> {
+    oidcLoginOptions = oidcLoginOptions as IAccessTokenOidcOptions;
```

Comment:
If we're expecting the additional properties from `IAccessTokenOidcOptions` here, shouldn't the parameter have that type? The cast seems dangerous...

```suggestion
  async handle(oidcLoginOptions: IAccessTokenOidcOptions): Promise<ISolidSession> {
```

Reply:
I guess the cast is dangerous if you don't perform a test before-hand. I added a test.